packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -60,11 +60,12 @@ export default class Classifier extends React.Component {
   }
 
   componentDidMount () {
-    const { onAddToCollection, onCompleteClassification, onToggleFavourite, project } = this.props
+    const { onAddToCollection, onCompleteClassification, onToggleFavourite, project, subjectSetID, workflowID } = this.props
     this.setProject(project)
     this.classifierStore.setOnAddToCollection(onAddToCollection)
     this.classifierStore.classifications.setOnComplete(onCompleteClassification)
     this.classifierStore.setOnToggleFavourite(onToggleFavourite)
+    this.classifierStore.workflows.selectWorkflow(workflowID, subjectSetID)
```

Comment:
I think I would prefer on continuing to rely the reactive paradigm we have setup using observers in the stores for asynchronous request or setup of resources in the store. However, I think that will take some additional refactoring that is out of scope here.

Reply:
I couldn't come up with a good way to pass the classifier props down into the workflow store, except by calling an action directly. If this block could be replaced with `store.create(snapshot)`, that would be great.